src/datta_lab_to_nwb/markowitz_gillis_nature_2023/markowitz_gillis_nature_2023_metadata.yaml

```diff
@@ -67,3 +69,121 @@ Behavior:
       9: Reared sniff
       27: Pause
       14: Body lick
+    id2sorted_index: # Mapping from syllable id --> syllable index (sorted by usage) [See syllable_stats_offline.toml]
+      74: 0
+      20: 1
+      22: 2
+      17: 3
+      86: 4
+      8: 5
+      12: 6
+      91: 7
+      27: 8
+      64: 9
+      95: 10
+      71: 11
+      19: 12
+      44: 13
+      39: 14
+      25: 15
+      65: 16
+      58: 17
+      41: 18
+      47: 19
+      53: 20
+      57: 21
+      94: 22
+      76: 23
+      35: 24
+      92: 25
+      72: 26
+      7: 27
+      5: 28
+      77: 29
+      67: 30
+      33: 31
+      66: 32
+      42: 33
+      15: 34
+      30: 35
+      56: 36
+      11: 37
+      45: 38
+      13: 39
+      73: 40
+      62: 41
+      31: 42
+      59: 43
+      10: 44
+      75: 45
+      4: 46
+      54: 47
+      43: 48
+      26: 49
+      60: 50
+      85: 51
+      29: 52
+      80: 53
+      68: 54
+      38: 55
+      63: 56
+      -5: 57
+    sorted_index2id:
```

Comment:
I would not include a sorted version of a separate dictionary in the same file, since it can be done on the fly

Reply:
Whoops, sorry. These dictionaries should only be in the `labeled_events` branch/PR